DevCycle/DVCVariable.swift

```diff
@@ -52,6 +52,10 @@ public class DVCVariable<T> {
         self.type = variable.type
         self.evalReason = variable.evalReason
         self.isDefaulted = false
+        
+        if let handler = self.handler {
+            handler(self.value)
```

Comment:
Are there any tests we can add?

Reply:
I have some tickets in the backlog to tackle more integration type tests